config/routes.rb

```diff
@@ -1,6 +1,17 @@
+# frozen_string_literal: true
+
 Rails.application.routes.draw do
+  # devise_for :users
```

Comment:
[OPTIONAL] It would be nicer to remove the commented code so your code is clean.

Reply:
Thanks @medaminedev66, I will makesure to remove those codes